Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:alpine3.12 AS builder
+WORKDIR /go/src/github.com/testcontainers/helloworld
+COPY . ./
```

Comment:
this would prevent any possible caching

Consider using something like:
```dockerfile
COPY go.mod go.sum ./
RUN go mod download
```

Reply:
Aha - good call. Will do.
Thanks!